apps/kevin-malone/bot/common/threads/onboarding.py

```diff
@@ -73,7 +71,22 @@ async def control_hook(self, message, user_id):
                 user_id, self.cache, USER_DB_ID_CACHE_KEY, user["id"]
             )
             return StepKeys.ASSOCIATE_EXISTING_USER_WITH_GUILD.value
-        return StepKeys.PROMPT_USER_WALLET_ADDRESS.value
+        return StepKeys.PROMPT_USER_CONNECT_WALLET.value
+
+
+class PromptUserToConnectWallet(BaseStep):
+    """Step to submit wallet address for the govrn profile"""
+
+    name = StepKeys.PROMPT_USER_CONNECT_WALLET.value
+
+    msg = (
+        f"It doesn't look like your wallet is connected to this discord account please connected them at this link {constants.Bot.govrn_discord_auth_url}.\n\n"
```

Comment:
The new `/signature` route that we'll have in place for supporting this flow takes in an optional query param for the display name as `displayName` so we could add that to this link as well, such as:

`/signature?displayName=km-user-flow` -- the Signature flow uses this display name (if it's there) when creating a new user